internal/pkg/skuba/deployments/ssh/kubelet.go

```diff
@@ -163,6 +163,14 @@ func kubeletConfigure(t *Target, data interface{}) error {
 			return err
 		}
 	}
+	if _, err := os.Stat(skuba.VSphereCloudConfFile()); err == nil {
```

Comment:
`else if` or change to use `switch` for multiple conditions?

Reply:
done